README.md

```diff
@@ -1,5 +1,29 @@
 # OpenStax Kinetic  - a Research Platform for Education
 
+## Getting Started
+
+To get started make sure you have yarn, docker, and docker-compose installed. Next try running these commands in the root directory:  
+
+```bash
+$> docker compose build
+$> docker compose up
+```
+You may run into migration issues at first. To fix that go into the back-end terminal with:
+
+```bash
+$> docker compose exec api /bin/bash
+```
+
+Then in that terminal run:
+
+```bash
+$> bin/rails db:migrate RAILS_ENV=development
+```
+
+and try running the docker compose commands again to see if that fixes it.
+
+When run correctly the front-end may be at http://localhost:4000/.
+
 ## Frontend
 
 To run the front-end for development:
```

Comment:
We can probably create a separate "README.md" file in the `frontend` directory and keep some of the frontend specific stuff like this in there

This section tripped my up when I first started, as I had assumed we needed the frontend running in a separate terminal. Since the docker container runs the FE for us, this is mostly irrelevant (I am trying to think of a situation where we'd run the FE without the BE)